README.md

```diff
@@ -66,6 +92,15 @@ Once the all dependencies have been installed and secret values added, start the
 $ poetry run flask run
 ```
 
+If you get an error saying 
+
+> Recreating virtualenv todo-app in \<your working directory\>\\.venv
```

Comment:
Why does this happen?

I also noticed my integration tests breaking if I run `poetry run pytest`, and the thing that fixes it is simply updating the file (e.g. adding a character and deleting it) and then trying again. How can I fix this?

Reply:
Adding a character and deleting it as a solution makes me think we've got a caching error - for example potentially we want to add `**/__pycache__` into your `.dockerignore` file to prevent the os-specific cache files leaking between the two environments

I think that's related to your .venv breaking too - detail on the compose file but I'd suggest we _only_ map the code in